cwd/app.js

```diff
@@ -273,19 +273,23 @@
       characterText.textContent = view.message;
     }
 
-    // Testing functionality of length based font sized
-    let messageLength = characterText.innerText.trim().length;
+    const messageLength = characterText.innerText.trim().length;
+    const screenSize = document.getElementById('svg-wrap').width.baseVal.value;
+    var messageSize;
```

Comment:
Haha the realllyy functional way of doing this is to make `messageSize` set to the result of a function (that maybe returns in a switch statement).  Then you don't create a variable at all.  But, I don't need to subject you to these horrors.

For future reference though, use `let` rather than `var`.